### ISSUE TITLE
LambdaLR doesn't follow PyTorch's LRScheduler API

### ISSUE THREAD

**Issue by BelugaZone** [KaKituken/affordance-aware-any#7]
When training my model, I encountered the following error: 

"The provided lr scheduler LambdaLR doesn't follow PyTorch's LRScheduler API. You should override the LightningModule.lr_scheduler_step hook with your own logic if you are using a custom LR scheduler." 

I'm using PyTorch Lightning version 1.6.1.
